src/test/java/com/google/api/codegen/testdata/python_test_library.baseline

```diff
@@ -51,6 +57,23 @@ class TestLibraryServiceClient(unittest2.TestCase):
 
         grpc_stub.CreateShelf.assert_called_once()
 
+    @mock.patch('google.gax.config.API_ERRORS', (CustomException,))
+    @mock.patch('google.gax.config.create_stub')
+    def test_create_shelf_exception(self, mock_create_stub):
+        # Mock gRPC layer
+        grpc_stub = mock.Mock()
+        mock_create_stub.return_value = grpc_stub
+
+        client = library_service_client.LibraryServiceClient()
+
+        # Mock request
+        shelf = library_pb2.Shelf()
+
+        # Mock exception response
+        grpc_stub.CreateShelf.side_effect = CustomException()
+
+        self.assertRaises(errors.GaxError, client.create_shelf, shelf)
```

Comment:
That is not particularly easy to do with `unittest.assertRaises`. The correct way to do this stuff in Python is to have _subclasses_ of `GaxError` for each code (such as [what I almost did here][1]), which would then be what we would test for. But we do not right now, so what is here should be fine.

  [1]: https://github.com/googleapis/gax-python/pull/142#discussion_r94071999